niceman/resource/aws_ec2.py

```diff
@@ -58,6 +58,26 @@ class AwsEc2(Resource):
     _ec2_resource = attr.ib(default=None)
     _ec2_instance = attr.ib(default=None)
 
+    @staticmethod
+    def get_backend_properties():
+        """
+        Return the config properties specific to this resource type.
+
+        Returns
+        -------
+        dict : key = resource property, value = property description
+        """
+        return {
+            'access_key_id': 'AWS access key for remote access to your Amazon subscription.',
```

Comment:
IMHO those better live along within `attr.ib` definition, i.e. within its metadata field.  I think we might even benefit from
providing a tiny adapter
```python
def attrib(*args, **kwargs):
   doc = kwargs.get('doc')
   metadata = kwargs.get('metadata', {})
   if doc:
        metadata['doc'] = doc
   if metadata:
        kwargs['metadata'] = metadata
   return attr.ib(*args, **kwargs)
```

then we could do 
```python
access_key_id = attrib(
   default=None,
   doc="AWS secret access key for remote access to your Amazon subscription")
```